src/main/java/frc/robot/RobotContainer.java

```diff
@@ -49,7 +50,8 @@ public class RobotContainer {
     private final JoystickButton yButton = new JoystickButton(mainController, XboxController.Button.kY.value),
             xButton = new JoystickButton(mainController, XboxController.Button.kX.value),
             bButton = new JoystickButton(mainController, XboxController.Button.kB.value),
-            aButton = new JoystickButton(mainController, XboxController.Button.kA.value);
+            aButton = new JoystickButton(mainController, XboxController.Button.kA.value),
+            startButton = new JoystickButton(mainController, XboxController.Button.kStart.value);
```

Comment:
lol are we going to leave this in even though we couldn't figure out which button was start

Reply:
🤷 doesn't really matter